webpack.config.js

```diff
@@ -1,41 +1,34 @@
 const path = require('path');
 const webpack = require('webpack');
 
-module.exports = (env = {}) => {
-  return {
-    devtool: (() => {
-      if (!env.production) {
-        return 'cheap-eval-source-map';
-      }
-    })(),
-    entry: path.resolve(__dirname, 'electron/renderer/src/index.js'),
-    module: {
-      rules: [
-        {
-          exclude: /(node_modules)/,
-          test: /\.js$/,
-          use: ['babel-loader'],
+module.exports = (env = {}) => ({
+  devtool: env.production ? '' : 'cheap-eval-source-map',
+  entry: path.resolve(__dirname, 'electron/renderer/src/index.js'),
+  mode: env.production ? 'production' : 'development',
```

Comment:
What do you think about inverting the condition to make production the default in case of an error?

Reply:
Good idea